From 3d7c35753eb328f1423bac43f0c7fe59fd3766d9 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Mat=C3=ADas=20Hern=C3=A1ndez?= <matiashe@google.com>
Date: Fri, 3 Jan 2025 13:38:10 +0100
Subject: [PATCH] Remove incorrect call to Window.addFlags with a system flag

The same flag is being (correctly) added in onCreate, so on top of being wrong this was also completely unnecessary.

Bug: 368579654
Test: manual
Flag: EXEMPT bugfix

(cherry picked from commit e0510a62eba04140e459cdd8232f35087aa3073b)
(cherry picked from https://googleplex-android-review.googlesource.com/q/commit:4f4d00f42271df5a2727cac373412dd0114076cf)
Merged-In: I539fca77a1d136bdd0e4892993020884430b8a4e
Change-Id: I539fca77a1d136bdd0e4892993020884430b8a4e
---
 .../NotificationAccessConfirmationActivity.java   | 15 ---------------
 1 file changed, 15 deletions(-)

diff --git a/src/com/android/settings/notification/NotificationAccessConfirmationActivity.java b/src/com/android/settings/notification/NotificationAccessConfirmationActivity.java
index 74b8102ee2..253ae88e4b 100644
--- a/src/com/android/settings/notification/NotificationAccessConfirmationActivity.java
+++ b/src/com/android/settings/notification/NotificationAccessConfirmationActivity.java
@@ -42,7 +42,6 @@ import android.os.UserManager;
 import android.service.notification.NotificationListenerService;
 import android.text.TextUtils;
 import android.util.Slog;
-import android.view.WindowManager;
 import android.view.accessibility.AccessibilityEvent;
 import android.widget.Toast;
 
@@ -160,20 +159,6 @@ public class NotificationAccessConfirmationActivity extends Activity
         getWindow().setCloseOnTouchOutside(false); 
     }
 
-    @Override
-    public void onResume() {
-        super.onResume();
-        getWindow().addFlags(
-                WindowManager.LayoutParams.SYSTEM_FLAG_HIDE_NON_SYSTEM_OVERLAY_WINDOWS);
-    }
-
-    @Override
-    public void onPause() {
-        getWindow().clearFlags(
-                WindowManager.LayoutParams.SYSTEM_FLAG_HIDE_NON_SYSTEM_OVERLAY_WINDOWS);
-        super.onPause();
-    }
-
     private void onAllow() {
         mNm.setNotificationListenerAccessGranted(mComponentName, true);
 
-- 
2.48.1.262.g85cc9f2d1e-goog

